.github/workflows/run_tests.yml

```diff
@@ -8,9 +8,25 @@ on:
   push:
     branches:
       - master
+    paths:
+      - 'ivadomed/**'
+      - 'testing/**'
+      - '.github/workflows/run_tests.yml'
+      - 'pytest.ini'
+      - 'environment.yml'
+      - 'requirements.txt'
+      - 'setup.py'
```

Comment:
Sure, that'll be safer. If people start getting annoyed at the tests running too much it's hopefully obvious where to add them.

I think there's a few more paths to add to this though, and I want to avoid running tests on unrelated workflows -- not that we edit workflows very often, but it would be against the spirit of the PR.

```suggestion
    paths-ignore
      - '**.md'
      - '**.rst'
      - '**.bib':
      - '.github/**'
      - '!.github/workflows/run_tests.yml'
      - 'dev/**'
      - 'docs/**'
      - 'images/**'
      - '.gitignore'
      - '.pre-commit-config.yaml'
      - '.readthedocs.yml'
```

(I'm not sure if the `!` is going to work right there to make it so that *only* run_tests.yml triggers, I'm just trying to [read the docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet).)